models/bookings.js

```diff
@@ -0,0 +1,13 @@
+const mongoose = require('mongoose');
+
+const bookingSchema = new mongoose.Schema({
+  id_room: ObjectId,
+  start: Date,
+  end: Date,
+  sit: Boolean,
+  video: Boolean,
+  number_people: Number,
+  status: Boolean,
+});
```

Comment:
Is user_id missing ?

Reply:
Also, created_at is maybe an automatic field in mongo, is that why it is not declared in the model ?

Reply:
j'vois tjr pas user_id whallah